portal/account/account.lib.php

```diff
@@ -0,0 +1,269 @@
+<?php
+session_start();
+if ($_SESSION['patient_portal_onsite_two'] && $_SESSION['pid']) {
+    $ignoreAuth_onsite_portal_two = true;
+}
+
+require_once("../../interface/globals.php");
+include_once("$srcdir/patient.inc");
+require_once(dirname(__FILE__) . "/../lib/portal_mail.inc");
+require_once("$srcdir/pnotes.inc");
+
+$action = isset($_REQUEST['action']) ? $_REQUEST['action'] : '';
+
+if ($action == 'set_lang') {
+    $_SESSION['language_choice'] = (int) $_REQUEST['value'];
+    echo 'okay';
+    exit();
+}
+
+if ($action == 'get_newpid') {
+    $rtn = is_new($_REQUEST['dob'], $_REQUEST['last'], $_REQUEST['first']);
+    if ((int) $rtn != 0) {
+        echo xlt("This account already exists.") . "\r\n\r\n" . xlt("If you are having troubles logging into your account.") . "\r\n" . xlt("Please contact your provider.") . "\r\n" .
+             xla("Reference this Account Id: ") . $rtn;
```

Comment:
use xlt()

Note xla (and attr() ) is for use in html attributes where quotes can mess things up.